data/json/items/vehicle/engine.json

```diff
@@ -0,0 +1,137 @@
+[
+  {
+    "abstract": "engine_combustion",
+    "type": "ENGINE",
+    "category": "veh_parts",
+    "name": "internal combustion engine",
+    "material": "iron",
+    "symbol": ":",
+    "color": "light_cyan"
+  },
+  {
+    "abstract": "engine_diesel",
+    "copy-from": "engine_combustion",
+    "type": "ENGINE",
+    "name": "base diesel engine",
+    "faults": [ "fault_engine_belt_drive", "fault_engine_glow_plug", "fault_engine_immobiliser", "fault_engine_pump_diesel", "fault_engine_pump_water", "fault_engine_starter" ]
+  },
+  {
+    "abstract": "engine_gasoline",
+    "copy-from": "engine_combustion",
+    "type": "ENGINE",
+    "name": "base gasoline engine",
+    "faults": [ "fault_engine_belt_drive", "fault_engine_immobiliser", "fault_engine_pump_fuel", "fault_engine_pump_water", "fault_engine_starter" ]
+  },
+  {
+    "id": "1cyl_combustion",
+    "copy-from": "engine_gasoline",
+    "type": "ENGINE",
+    "name": "1-cylinder engine",
+    "description": "A single-cylinder 4-stroke combustion engine.",
+    "weight": 20000,
+    "volume": 6,
+    "price": 10000,
+    "displacement": 60,
+    "faults": [ "fault_engine_starter" ]
```

Comment:
It replaces the other `faults`. I want to eventually model the 1-cyl 2-stroke starting with a pull cord adn the 1-cyl 4-stroke having all integral parts apart from an external starter.